### ISSUE TITLE
blackbuntu.org is offline?

### ISSUE THREAD

**Issue by andrewpayne68** [blackbuntu/system#2]
I noticed that the blackbuntu.org repo and web site have been offline for a couple of weeks now?

**Comment by SandroCristiano** [comment]
Yeah. Their twitter accounts haven't had any updates since April. The last push to this git was in May and both their pages on linkedin are down. Pick your poison, have they: 
... gotten full time jobs and abandoned the project;
... fled the country;
... been arrested; 
... been killed.

**Comment by TheArmChairDetective** [comment]
It's still offline as of 05/10/2023 and the url for the .org is for sale at over $5k. It's a shame as I prefernot to follow the kali herd and be somewhat independant.   A great loss, whichever of the above options is the right one.
I wish I had the skill set to pick the project up.

**Comment by TheArmChairDetective** [comment]
> Yeah. Their twitter accounts haven't had any updates since April. The last push to this git was in May and both their pages on linkedin are down. Pick your poison, have they: ... gotten full time jobs and abandoned the project; ... fled the country; ... been arrested; ... been killed.

One more option.

Got paid to make it go away :-)

**Comment by HarshShinde0** [comment]
isnt there any source where Blackbuntu 22.04.2 LTS amd64 is available ??

**Comment by TheArmChairDetective** [comment]
Sorry, im a potential user and have no connection to it
JC

Sent with [Proton Mail](https://proton.me/) secure email.
